package.json

```diff
@@ -43,6 +43,7 @@
   },
   "dependencies": {
     "babel-eslint": "10.0.1",
+    "eslint-plugin-babel": "^5.3.0",
```

Comment:
`eslint-plugin-babel` was renamed to `@babel/eslint-plugin`, please update this to use that one instead.